app/controllers/AiModelController.scala

```diff
@@ -57,6 +57,16 @@ object UpdateAiModelParameters {
   implicit val jsonFormat: OFormat[UpdateAiModelParameters] = Json.format[UpdateAiModelParameters]
 }
 
+case class RegisterAiModelParameters(id: ObjectId, // must be a valid MongoDB ObjectId
```

Comment:
Can this be validated in registerAiModel?

Reply:
It’s automatically validated by the json reads of ObjectId. I just added the comment as a hint for writing the client.